### PR TITLE
Discord Submissions (27 apr 2020)

### PR DESCRIPTION
**Pull request by barsoosayque**
New stuff:

- A bunch of sewing items (by *bi*)
-  :spider_web: Arachnid mutations (by *chronchitis*)

### CODE REVIEW

File: gfx/UltimateCataclysm/pngs_normal_32x32/items/needles/needles.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "bg": []
+  },
+  {
+    "id": "needle_wood",
+    "fg": [
+      "needle_wood"
+    ],
+    "bg": []
+  }
+]
```
Comment (anothersimulacrum) [inline gfx/UltimateCataclysm/pngs_normal_32x32/items/needles/needles.json:23]:
```suggestion
]

```

File: gfx/UltimateCataclysm/pngs_normal_32x32/items/string_36/string_36.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "string_36",
+  "fg": [
+    "string_36"
+  ],
+  "bg": []
+}
```
Comment (anothersimulacrum) [inline gfx/UltimateCataclysm/pngs_normal_32x32/items/string_36/string_36.json:7]:
```suggestion
}

```

File: gfx/UltimateCataclysm/pngs_normal_32x32/items/string_6/string_6.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "string_6",
+  "fg": [
+    "string_6"
+  ],
+  "bg": []
+}
```
Comment (anothersimulacrum) [inline gfx/UltimateCataclysm/pngs_normal_32x32/items/string_6/string_6.json:7]:
```suggestion
}

```

File: gfx/UltimateCataclysm/pngs_normal_32x32/items/thread/thread.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "thread",
+  "fg": [
+    "thread"
+  ],
+  "bg": []
+}
```
Comment (anothersimulacrum) [inline gfx/UltimateCataclysm/pngs_normal_32x32/items/thread/thread.json:7]:
```suggestion
}

```

File: gfx/UltimateCataclysm/pngs_tall_32x64/overlay/mutations/arachnid_arms/arachnid_arms.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    "bg": []
+  },
+  {
+    "id": "overlay_male_mutation_ARACHNID_ARMS",
+    "fg": [
+      "arachnid_arms_m"
+    ],
+    "bg": []
+  }
+]
```
Comment (anothersimulacrum) [inline gfx/UltimateCataclysm/pngs_tall_32x64/overlay/mutations/arachnid_arms/arachnid_arms.json:16]:
```suggestion
]

```

File: gfx/UltimateCataclysm/pngs_tall_32x64/overlay/mutations/chitin/chitin.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    "bg": []
+  },
+  {
+    "id": "overlay_male_mutation_CHITIN",
+    "fg": [
+      "chitin_m"
+    ],
+    "bg": []
+  }
+]
```
Comment (anothersimulacrum) [inline gfx/UltimateCataclysm/pngs_tall_32x64/overlay/mutations/chitin/chitin.json:16]:
```suggestion
]

```

File: gfx/UltimateCataclysm/pngs_tall_32x64/overlay/mutations/chitin2/chitin2.json [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    "bg": []
+  },
+  {
+    "id": "overlay_male_mutation_CHITIN2",
+    "fg": [
+      "chitin2_m"
+    ],
+    "bg": []
+  }
+]
```
Comment (anothersimulacrum) [inline gfx/UltimateCataclysm/pngs_tall_32x64/overlay/mutations/chitin2/chitin2.json:16]:
```suggestion
]

```

**Comment by barsoosayque** [comment]
ah, I really need to fix my ranger script to put those eof newlines

**Pull request merged**